src/lib.rs

```diff
@@ -395,7 +395,7 @@ pub fn create(value: &EtherValue, data: &[u8]) -> CreateResult {
     };
 
     match ret {
-        0 => CreateResult::Successful(result),
+        0 => CreateResult::Successful(Address { bytes: result }),
```

Comment:
This is nicer, but I'd feel safer if `let mut result` would be changed to address above.

Reply:
done